lib/bibdata_rs/src/solr/ephemera_solr_mapping.rs

```diff
@@ -154,6 +155,21 @@ mod tests {
     }
 
     #[test]
+    fn it_has_the_provenance_from_the_ephemera_folder_item() {
+        let ephemera_item = EphemeraFolderItem::builder()
+            .id("abc123".to_owned())
+            .title(vec!["Our favorite book".to_owned()])
+            .provenance("Test name".to_owned())
+            .build()
+            .unwrap();
+        let solr_document = SolrDocument::from(&ephemera_item);
+        assert_eq!(solr_document.id, "abc123");
+        assert_eq!(
+            solr_document.provenance_display,
+            Some("Test name".to_owned())
+        );
+    }
+
```

Comment:
```suggestion
#[test]
```

It looks like the `#[test]` attribute is missing from the next test now